src/genbenchQC/evaluate_splits.py

```
@@ -105,52 +90,103 @@ def run(train_files, test_files, format,
+            filter_fasta_by_ids(test_fasta_path, new_test_fasta_path, set(results_filt["query"]))
+            filter_fasta_by_ids(train_fasta_path, new_train_fasta_path, set(results_filt["target"]))
+            
+            # Get basic stats for HTML report
+            basic_stats = get_basic_stats(train_filenames, train_sequences, test_filenames, test_sequences)
+
+            # Generate HTML report
+            generate_splits_html_report(basic_stats, threshold_stats, results, results_filt, html_report_path, plots_dir)
+
+            logging.info("Train-test split evaluation successfully completed.")
+    except Exception:
+        logging.exception(f"Train-test split evaluation failed. ")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)